bodhi/server/push.py

```diff
@@ -157,5 +147,40 @@ def push(username, cert_prefix, config, **kwargs):
             force=True,
         )
 
+
+def _filter_releases(session, query, releases=None):
+    """
+    Apply a filter() transformation to the given query on Updates to filter updates that match the
+    given releases argument. If releases evaluates "Falsey", this function will filter for Updates
+    that are part of a current Release.
+
+    :param session:  The database session
+    :param query:    An Update query that we want to modify by filtering based on Releases
+    :param releases: A comma-separated string of release names
+
+    :returns:        A filtered version of query with an additional filter based on releases.
+    """
+    # We will store models.Release object here that we want to filter by
+    _releases = []
+
+    if releases:
+        for r in releases.split(','):
+            release = session.query(Release).filter(
+                or_(Release.name == r,
+                    Release.name == r.upper(),
+                    Release.version == r)).first()
+            if not release:
+                click.echo('Unknown release: %s' % r)
+                sys.exit(1)
+            else:
+                _releases.append(release)
+    else:
+        # Since the user didn't ask for specific Releases, let's just filter for releases that are
+        # current.
+        _releases = session.query(Release).filter(Release.state == ReleaseState.current)
```

Comment:
@puiterwijk I don't think ```!=archived``` is what we want either, because there is also a ```disabled``` state. It seems that the four states are ```disabled```, ```pending``` (which might be the same as what you call Development?), ```current```, and ```archived```.